.github/workflows/ci.yml

```diff
@@ -0,0 +1,53 @@
+name: Run tests
+
+on:
+  push:
+    paths-ignore:
+      - 'LICENSE.md'
+      - 'README.md'
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+    tags: '*'
+
+jobs:
+  test:
+    runs-on: ${{ matrix.os }}
+    if: "!contains(github.event.head_commit.message, 'skip ci')"
```

Comment:
To allow failures on nightly the line `continue-on-error: ${{ matrix.version == 'nightly' }}` could be added here

Reply:
Good catch! Thanks a lot!